### ISSUE TITLE
Vision for SUIR?

### ISSUE THREAD

**Issue by PaulAtST** [Semantic-Org/Semantic-UI-React#4230]
Hi, long story short I stared with a google searching how to update the font awesome icon set in Semantic UI React, and came across the issues talking about how Semantic UI React might move forwards since the Semantic UI repo is no longer updated. 

This got me wondering is there a vision for how Semantic UI React will eventually move forward? Such as fork the Semantic UI project, support the Fomantic UI, make a Fomantic UI React project, or maybe another path I haven't found people talking about? 

This is a great UI toolkit, and I feel like creating a vision for it would help people contribute.

**Comment by sysrage** [comment]
I would also love to know what's going on with this project. It seems that SUIR v3 efforts have been stalled for awhile (https://github.com/Semantic-Org/Semantic-UI-React/pull/4233). Do we all need to move on? 😢

**Comment by sunech** [comment]
Agree. This is a really great project and the work that has been put in to this is really impressive. Despite looking at newer and more active alternatives, I haven't been able to find anything as overall appealing to me as SUIR.

Would be a great loss if it slowly dies.

**Comment by jkytomak** [comment]
Any news about this?
We have been using fomantic-ui with semantic-ui-react for 4 years now.
But looks like there is no future for semantic-ui project and owner of it has toxic attitude against fomantic:
https://github.com/Semantic-Org/Semantic-UI/issues/7078#issuecomment-1270568704

We are missing all the new things from Fomantic and have to start planning shifting away from the framework to something else if this does not get sorted.

Fomantic-ui is still quite compatible with semantic-ui-react so forking it to fomantic-ui-react would be a straightforward thing and then could start slowly adding support for new features from fomantic.

ps. Thanks for maintaining this repo, too bad the base library is not maintained.

**Comment by layershifter** [comment]
@jkytomak it would be great to push SUIR to v3 (see #4409) and fork to be FUIR as FUI is not fully compatible with SUI anymore. However, at this point I _personally_ don't have capacity and, let's be honest, enough motivation to push it forward 😢

**Comment by felixmosh** [comment]
@layershifter can you add me as a maintainer? I would like to contribute and try to push this project forward.
I think that the main issue is the coupling to the css... maybe we can fork it & move it forward... fix all the css issues that it has (In my project there is a file with all the fixes)

**Comment by layershifter** [comment]
@felixmosh The original intent of SUIR was to remain CSS-agnostic and rely entirely on CSS from Semantic UI. The issue with that approach is pretty clear - Semantic UI is no longer maintained 🪦

The Fomantic UI team seems to be doing a decent job, but they've introduced changes that make their version incompatible with SUI, see [#4021](https://github.com/Semantic-Org/Semantic-UI-React/pull/4021). Having Semantic UI React no longer work with Semantic UI is a breaking change that defeats its purpose.  

My initial plan was to bring SUIR to a better state by:  
- Updating it for React 18 and Strict Mode support
- Releasing a v3 version to clean up legacy issues
- Migrating the documentation to a modern framework like Next.js or Remix

From there, the idea was to fork and create Fomantic UI React, as Fomantic UI resolves many of Semantic UI's existing problems. However, this [comment](https://github.com/Semantic-Org/Semantic-UI-React/issues/4230#issuecomment-2275315806) complicates things 🥹.  

While a fork seems inevitable, I'd like to ensure we stay true to SUIR’s original philosophy. Does it make sense?

**Comment by felixmosh** [comment]
Thank you for your explanation & vision....🙏🏽
Why the semantic-ui (css wise) is not passed to someone that whats to maintain it?

**Comment by layershifter** [comment]
> Why the semantic-ui (css wise) is not passed to someone that whats to maintain it?

@felixmosh that's above my pay grade 😸 Jokes aside, these issues contain some information:
- https://github.com/Semantic-Org/Semantic-UI/issues/6109
- https://github.com/Semantic-Org/Semantic-UI/issues/6413

P.S. To be clear, neither me of @levithomason ever had access to the main repo.
